tests/yam/validate/validate_installed_patterns.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
```

Comment:
Use the content of this file already reviewed:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21550/files#diff-0c3c7b906c2b6f8ca97690eb7bd8b81cb039736b0ca72c717a366f0eb820d77aR1-R26